package.json

```diff
@@ -26,31 +26,31 @@
   },
   "homepage": "https://github.com/LibertyDSNP/schemas#readme",
   "dependencies": {
-    "@frequency-chain/api-augment": "0.0.0-45e306",
+    "@frequency-chain/api-augment": "^1.8.1",
     "@polkadot/api": "^10.7.3",
     "json-stringify-pretty-compact": "^4.0.0",
     "ts-node": "^10.9.1",
-    "typescript": "^5.0.4"
+    "typescript": "^5.2.2"
   },
   "optionalDependencies": {
     "@dsnp/parquetjs": "^1.3.0"
   },
   "devDependencies": {
-    "@dsnp/parquetjs": "^1.3.0",
+    "@dsnp/parquetjs": "^1.3.4",
     "@dsnp/test-generators": "^0.1.0",
     "@polkadot/types": "^10.7.3",
-    "@types/jest": "^29.5.2",
-    "@typescript-eslint/eslint-plugin": "^5.59.8",
-    "@typescript-eslint/parser": "^5.59.8",
-    "avro-js": "^1.11.1",
-    "eslint": "^8.41.0",
-    "eslint-config-prettier": "^8.8.0",
-    "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-jest": "^27.2.1",
-    "eslint-plugin-prettier": "^4.2.1",
-    "jest": "^29.5.0",
-    "jest-environment-jsdom": "^29.5.0",
-    "prettier": "^2.8.8",
-    "ts-jest": "^29.1.0"
+    "@types/jest": "^29.5.8",
+    "@typescript-eslint/eslint-plugin": "^6.11.0",
+    "@typescript-eslint/parser": "^6.11.0",
+    "avro-js": "^1.11.3",
+    "eslint": "^8.53.0",
+    "eslint-config-prettier": "^9.0.0",
```

Comment:
This works, but heads up that with the [changes in eslint 9.0 coming](https://eslint.org/blog/2023/11/whats-coming-in-eslint-9.0.0/) there will be separation between the use of eslint and prettier